iep-module-dynconfig/src/main/resources/reference.conf

```diff
@@ -0,0 +1,6 @@
+
+netflix.iep.archaius {
+  url = "http://localhost:7101/props"
```

Comment:
In the local development case, if no override is provided for this value, then the DynamicConfigManager will read its own configuration and use that as the override, correct?

I don't think this should be an issue and it's easy enough to point it at an iep-archaius instance in the cloud.

Reply:
I don't usually include the module when working locally, it is only needed when running in the cloud. For this module I added an enabled flag and it is disabled by default.